### PR TITLE
Update FML maps to fix breaking changes due to updated CDA logical Model 

### PR DESCRIPTION
**Pull request by robeastwood-agency**
Refer ticket #56 

This PR introduces a number of changes to align with the newly updated Core CDA Logical Model 2.0.0-sd-snapshot1.

### FHIR to CDA maps
This has been my focus given my experience is solely in this area.

#### Changes
The changes include:
* updated canonical URIs for all of the target CDA logical models
* introduction of the choice group `item` in a number of places
    * AddressAD datatype group
    * HumanNameEN datatype group
    * a number of `name` elements such as Organization name and various medicine related names
* replace rule for XML inner text of `.data` with `.xmlText` 
* Update sample FHIR Bundle instance so that it can be transformed without error (section.text to use `<span>` instead of `<i>` tag)

#### Testing
* all updated map files were successfully loaded to the matchbox server without error, using the steps outlined in [fhirtocda.http](https://github.com/hl7ch/cda-fhir-maps/blob/master/fhirtocda.http) (server rather than localhost)
* all 4 transforms in the same .http file were successful to generate a full CDA instance as a response
* all of the remaining FHIR to CDA .http files were used to POST the remaining maps without issue and transforms were attempted but most resulted in errors such that profiles was unable to be located in order to validate it.

In summary, it looks good, but if it were possible for someone to load the relevant medication profiles, I could repeat the transforms to ensure there are no other map rules needing attention.

### CDA to FHIR maps

I did start down this path, but quickly realised that I was out of my depth. 

The canonical URIs of the source CDA logical models were easily changed as above but POSTing to the matchbox server resulted in complaints about not expecting "as queried" or "as produced" instead of "as source" / "as target". Being unfamiliar with this content I decided against making any changes in this PR for CDA to FHIR mapping.

Creating a draft PR in order to seek feedback etc.

### CODE REVIEW

**Comment by oliveregger** [comment]
@robeastwood-agency wow that's a great PR, thanks! I will test it out and will adapt the CDA to FHIR maps

**Comment by oliveregger** [comment]
i merge it in and we continue from there ...

**Comment by robeastwood-agency** [comment]
Thanks @oliveregger; sounds good.

I have made a start on the CDA to FHIR maps to see how far I could get and am progressing quite well it seems. There are a lot of existing rules that need updating to eliminate server transform errors, which are not related to the recent core logical model updates.

For example [this](https://github.com/hl7ch/cda-fhir-maps/blob/master/input/maps/CdaToBundle.map#L52)
```
cda.id -> bundle.identifier "identifier";
```
needs to be changed to use the datatype rule for II, such as this:
```
cda.id as id -> bundle.identifier as identifier then II(id, identifier) "identifier";
```
And other similar type of rules to better use existing datatype rules, although I haven't yet got to a successful transform to make sure it all works etc.

I'd be happy to keep progressing down this path and submit a 2nd PR, albeit with a lot more changes, if it was OK with you.

**Comment by oliveregger** [comment]
I just ended up with the same error message, i'm puzzled ...

**Comment by oliveregger** [comment]
i'm not sure why this short syntax is not working anymore, it should actually
